definitions/npm/redux_v3.x.x/flow_v0.55.x-/test_createStore.js

```diff
@@ -5,9 +5,9 @@ import { createStore } from 'redux'
 type State = Array<number>;
 type Action = { type: 'A' };
 type Store = ReduxStore<State, Action>;
-const reducer = (state: State, action: Action): State => state
+const reducer = (state: State = [], action: Action): State => state
 
-//
+//s
```

Comment:
Aww, man, my `Cmd` key is on the fritz, so I'm leaving random "s" all over the place when I intend to save. I'll force push this away.